app/javascript/components/subject/MoveToHousehold.js

```diff
@@ -73,50 +205,82 @@ class MoveToHousehold extends React.Component {
           console.error(error);
         });
     });
+  };
+
+  /**
+   * Grabs a formatted string with the name of the patient who is being moved to a household.
+   */
+  getPatientName = () => {
+    return `${this.props.patient?.last_name || ''}, ${this.props.patient?.first_name || ''} ${this.props.patient?.middle_name || ''}`;
+  };
+
+  /**
+   * Handles a key press event on the search form control.
+   * Checks for enter button press and prevents submisson event.
+   * @param {Object} event
+   */
+  handleKeyPress(event) {
+    if (event.which === 13) {
+      event.preventDefault();
+    }
   }
 
-  createModal(title, toggle, submit) {
+  createModal(title, toggle) {
     return (
-      <Modal size="lg" show centered onHide={toggle}>
+      <Modal dialogClassName="modal-move-household" show centered onHide={toggle}>
         <Modal.Header>
           <Modal.Title>{title}</Modal.Title>
         </Modal.Header>
-        <Modal.Body>
+        <Modal.Body className="modal-move-household-body">
           <Form>
             <Row>
               <Form.Group as={Col}>
-                <Form.Label className="nav-input-label">Select The New Monitoree That Will Respond For The Current Monitoree</Form.Label>
-                <Form.Label size="sm" className="nav-input-label">
-                  Note: The current monitoree will be moved into the selected monitoree&apos;s household
-                </Form.Label>
-                <Form.Control as="select" className="form-control-lg" id="hoh_selection" onChange={this.handleChange} defaultValue={-1}>
-                  <option value={-1} disabled>
-                    --
-                  </option>
-                  {this.state?.dependents?.map((member, index) => {
-                    return (
-                      <option key={`option-${index}`} value={member.id}>
-                        {member.last_name}, {member.first_name} Age: {member.age}, State ID: {member.state_id}
-                      </option>
-                    );
-                  })}
-                </Form.Control>
+                <Form.Label className="nav-input-label">Select The New Monitoree That Will Respond For {this.getPatientName()}</Form.Label>
```

Comment:
This is more of a minor thing/something for PO team to consider (and ik the language was already like this before), but it seems weird to me that every word of this sentence is capitalized.

Reply:
Agreed, updated and checked with PO team.